src/org/mirah/builtins/string_extensions.mirah

```diff
@@ -39,4 +39,27 @@ class StringExtensions
       end
     end
   end
+  
+  # separate to to_i(), as to_i() (in order to maintain Ruby compatibility) could return a BigInteger,
+  # while to_int() always returns an int.
+  macro def to_int:int
```

Comment:
Yeah. After sleeping on it, I agree. I think that while Ruby has Integer(x) as the safe coercion method, since Integer(x) means something else in Mirah, we shouldn't overload it.